components/CovidMap.js

```diff
@@ -0,0 +1,677 @@
+import React from 'react';
+import Tabletop from 'tabletop';
+import MarkerClusterer from '@google/markerclustererplus';
+
+const customClusterIconClasses = [
+  'custom-clustericon-first',
+  'custom-clustericon-second',
+  'custom-clustericon-third',
+  'custom-clustericon-fourth',
+  'custom-clustericon-fifth',
+];
+
+const mapStyle = {
+  width: '100%',
+  paddingTop: '67%',
+};
+
+const formStyle = {
+  width: '50%',
+  float: 'left',
+};
+
+const legendStyle = {
+  width: '50%',
+  float: 'right',
+};
+
+const legendItemStyle = {
+  width: '100%',
+  marginBottom: '5px',
+};
+
+const legendTextStyle = {
+  width: '90%',
+  height: '20px',
+  float: 'left',
+  textAlign: 'right',
+};
+
+const legendIconStyle = {
+  minWidth: '30px',
+  height: '20px',
+  float: 'right',
+  display: 'inline-block',
+};
+
+const legendIconFirst = {
+  background: '#3167ae',
+  width: '20px',
+  height: '20px',
+  borderRadius: '50%',
+  display: 'block',
+  float: 'right',
+  marginLeft: '5px',
+  marginRight: '5px',
+  opacity: '80%',
+};
+
+const legendIconSecond = {
+  background: '#4c5151',
+  width: '20px',
+  height: '20px',
+  borderRadius: '50%',
+  display: 'block',
+  float: 'left',
+  marginLeft: '5px',
+  marginRight: '5px',
+  opacity: '80%',
+};
+
+const legendIconThird = {
+  background: '#ce2827',
+  width: '20px',
+  height: '20px',
+  borderRadius: '50%',
+  display: 'block',
+  float: 'left',
+  marginLeft: '5px',
+  marginRight: '5px',
+  opacity: '80%',
+};
+
+async function fetchAPI() {
+  const url = `${window.location.origin}/.netlify/functions/google_maps_params`;
+  const res = await fetch(url);
+  const params = await res.json();
+  // Load the Google Maps API
+  const script = document.createElement('script');
+  script.src = `https://maps.googleapis.com/maps/api/js?key=${params.env}&callback=resolveGoogleMapsPromise`;
```

Comment:
We should update this to use the `params.env` varaible to pick the api key we want to use, like:

```
script.src = `https://maps.googleapis.com/maps/api/js?key=${params.client[params.env]}&callback=resolveGoogleMapsPromise`;
```

Reply:
Done!